storage/pebble/registers_pruner.go

```diff
@@ -0,0 +1,326 @@
+package pebble
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/cockroachdb/pebble"
+	"github.com/rs/zerolog"
+
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/module"
+	"github.com/onflow/flow-go/module/component"
+	"github.com/onflow/flow-go/module/irrecoverable"
+)
+
+const (
+	DefaultPruneThreshold      = uint64(100_000)
+	DefaultPruneThrottleDelay  = 10 * time.Millisecond
+	DefaultPruneTickerInterval = 10 * time.Minute
+)
+
+// pruneIntervalRatio represents an additional percentage of pruneThreshold which is used to calculate pruneInterval
+const (
+	pruneIntervalRatio  = 0.1
+	deleteItemsPerBatch = 256
```

Comment:
how did you come up with this limit?

Reply:
This configuration is not final and should be adjusted for optimal performance after testing on a live network.
I have added comment for it.

I temporarily set `deleteItemsPerBatch` to the same value used by the execution data pruner: 
https://github.com/onflow/flow-go/blob/b93c60f2a3dff31d8f76711822d4873cdce4a0eb/module/executiondatasync/tracker/storage.go#L394-L395 

The `pruneIntervalRatio` is temporarily set to 10%, as suggested in a previous [comment](https://github.com/onflow/flow-go/issues/6068#issuecomment-2292292123) (
> then pruneInterval is pruneThreshold plus some amount (say 10%)
)